commands/inspect_builder_test.go

```diff
@@ -69,13 +70,11 @@ func testInspectBuilderCommand(t *testing.T, when spec.G, it spec.S) {
 
 				h.AssertContains(t, outBuf.String(), `Remote
 ------
-
-ERROR: some remote error
+some remote error
```

Comment:
We should try to preserve the `ERROR` prefix for errors if we can (everywhere, not just here)